### ISSUE TITLE
Initial setup fails

### ISSUE THREAD

**Issue by yuvalsuede** [react-boilerplate/react-boilerplate#2776]
## Description
Running `npm run setup` fails during installation. 

## Steps to reproduce
Run `npm run setup`

**Expected behavior**
After installing the missing module, it keeps failing on other parts. 

**Output**
```
> node ./internals/scripts/setup.js

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'compare-versions'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
```

## Versions

- React-Boilerplate: version": "4.0.0",
- Node/NPM:` v10.16.3 / 6.9.0 `
- OS: MacOS 10.14.6

**Comment by vineyardbovines** [comment]
Interesting find, I'll try and take a look at this soon @yuvalsuede -- please ping me in a couple of days if you don't hear from me, I'm just coming back from a long trip.

**Comment by julienben** [comment]
Hey @yuvalsuede,

This is a strange issue because it only seems to have happened for a few users. I can't reproduce it even on a clean setup. Nonetheless, I believe it will be fixed in PR #2783. I added `compare-versions` to the list of dependencies in the `presetup` script (line 29 of `package.json`).

**Comment by julienben** [comment]
Thanks for reporting! Will close this for now as I'm assuming you worked around it on your end.
